.github/vale/styles/Klaw/branding.yml

```diff
@@ -24,3 +24,5 @@ swap:
   "type[ -]?scripts?": TypeScript
   "(?i)ssl": SSL
   "(?<!Klaw )cluster api": Klaw Cluster API
+  "(?i)klaw core": Klaw Core
+  "(?i)klaw": Klaw
```

Comment:
Adding the "Klaw" rule to this file should not be part of this PR, could you remove that?